nvm.sh

```diff
@@ -260,12 +260,12 @@ if [ -z "${NVM_DIR-}" ]; then
   export NVM_DIR
 else
   # https://unix.stackexchange.com/a/198289
-  case $NVM_DIR in
+  case ${NVM_DIR} in
     *[!/]*/)
       NVM_DIR="${NVM_DIR%"${NVM_DIR##*[!/]}"}"
       export NVM_DIR
       nvm_err "Warning: \$NVM_DIR should not have trailing slashes"
-    ;;
+      ;;
```

Comment:
the `;;` should be aligned with the line containing the `)`, please revert this change throughout

Reply:
Google makes terrible styleguides in my experience - merely widely adopted ones (because "Google").

Reply:
I see it as a terminator for the matching pattern.

Either way, I prefer the existing style here.

Reply:
Reverted this part.